databricks/koalas/frame.py

```diff
@@ -1155,27 +1155,36 @@ def nunique(self, axis: int = 0, dropna: bool = True, approx: bool = False,
             Don’t include NaN in the count.
         approx: bool, default False
             If False, will use the exact algorithm and return the exact number of unique.
-            If True, it uses Spark's approximate algorithm, which is faster in most circumstances.
-            Note: this parameter is specific to Spark and is not found in pandas.
+            If True, it uses the HyperLogLog approximate algorithm, which is significantly faster
+            for large amount of data.
+            Note: this parameter is specific to Koalas and is not found in pandas.
         rsd: float, default 0.05
-            Maximum estimation error allowed. Just like ``approx`` this parameter is specific to
-            Spark.
+            Maximum estimation error allowed in the HyperLogLog algorithm.
+            Just like ``approx`` this parameter is specific to Koalas.
 
         Returns
         -------
         The number of unique values per column as a pandas Series.
 
         Examples
         --------
-        >>> ks.DataFrame({'A': [1, 2, 3], 'B': [np.nan, 3, np.nan]}).nunique()
+        >>> df = ks.DataFrame({'A': [1, 2, 3], 'B': [np.nan, 3, np.nan]})
```

Comment:
My original motivation to not reuse `df` was for the single tests to be more self-contained. But this works fine with me as well and is probably more consistent with the other doctests.